index.html

```diff
@@ -22,46 +22,55 @@ <h3 >
 		</nav>
 	</header>
 	<section class="headline">
-		<h1>
-			Hey there. I’m Yoseph ,<br>
-			<span class="header_green">I’m a software developer</span>
-		</h1>
-		<p>
-			I can help you build a product , feature or website Look through some of my work and experience! If you like
-			what you see and have a project you need coded,
-			don’t hestiate to contact me.
-		</p>
+		<nav>
+			<ul class="header-nav-desktop">
+				<li>Portfolio</li>
+				<li>About</li>
+				<li>Contact</li>
```

Comment:
- [ ] Good job so far :clap: , you made a beautiful project however, this button should be responsive , when you click on them , they should bring the user to the corresponding section :call_me_hand:. Kindly make sure you add that to improve the user experience.
### What you should do
```bash
<a href="#id_of_section"> Portfolio<a/>
```
![Screenshot from 2023-01-20 15-01-39](https://user-images.githubusercontent.com/80458199/213701689-6660d2a6-8f41-4715-a631-bf0099ad5866.png)